### ISSUE TITLE
xccdf2inspec_stub: ID is required but some checklists don't have them

### ISSUE THREAD

**Issue by potatoqualitee** [mitre/saf#272]
I am trying to convert the [VMware_vSphere_7.0_STIG_Readiness_Guide-V1R2.zip](https://github.com/vmware/dod-compliance-and-automation/blob/master/vsphere/7.0/docs/VMware_vSphere_7.0_STIG_Readiness_Guide-V1R2.zip) file included in VMware's dod repo but ran into a problem

```
> saf generate:xccdf2inspec_stub --input="C:\temp\VMware_vSphere_7.0_STIG_Readiness_Guide-V1R2\VMware_vSphere_7.0_ESXI_STIG_Readiness_Guide-xccdf.xml" --output=C:\temp\stub
    TypeError: Cannot read properties of undefined (reading '@_id')
```

A contributor to the repo [highlighted](https://github.com/vmware/dod-compliance-and-automation/issues/79) that the issue appears to originate with this line:

https://github.com/mitre/saf/blob/main/src/commands/generate/xccdf2inspec_stub.ts#L135

Is there a way to make that optional?

**Comment by aaronlippold** [comment]
We will take a look

**Comment by camdenmoors** [comment]
Should be fixed in #273. Can you please validate that the attached are what is expected: [InSpec Profiles - VMware_vSphere_7.0.zip](https://github.com/mitre/saf/files/8430055/InSpec.Profiles.-.VMware_vSphere_7.0.zip)

**Comment by potatoqualitee** [comment]
Thank you so much! This looks really nice to me though @rlakey may know more.
